docs/website/docs/reference/bindings/python.md

```diff
@@ -83,15 +83,16 @@ To use IREE's Python bindings, you will first need to install
       iree-base-runtime
     ```
 
-=== ":material-alert: Nightly releases"
+=== ":material-alert: Nightly pre-releases"
 
-    Nightly releases are published on
+    Nightly pre-releases are published on
     [GitHub releases](https://github.com/iree-org/iree/releases).
 
     ``` shell
     python -m pip install \
       --find-links https://iree.dev/pip-release-links.html \
       --upgrade \
+      --pre \
```

Comment:
I think I caught most places across the repo that were using the old package names or were missing `--pre`. A few that I'm intentionally ignoring are:

* https://github.com/iree-org/iree/blob/43b22dec3e065d47a8c98507071691d52c7bfba8/experimental/distributed/python/pyproject.toml#L1-L6
* https://github.com/iree-org/iree/blob/43b22dec3e065d47a8c98507071691d52c7bfba8/integrations/pjrt/requirements.txt#L1-L4